src/plotman/archive.py

```diff
@@ -60,11 +66,17 @@ def get_archdir_freebytes(arch_cfg):
     return archdir_freebytes
 
 def rsync_dest(arch_cfg, arch_dir):
-    rsync_path = arch_dir.replace(arch_cfg['rsyncd_path'], arch_cfg['rsyncd_module'])
-    if rsync_path.startswith('/'):
-        rsync_path = rsync_path[1:]  # Avoid dup slashes.  TODO use path join?
-    rsync_url = 'rsync://%s@%s:12000/%s' % (
-            arch_cfg['rsyncd_user'], arch_cfg['rsyncd_host'], rsync_path)
+    arch_mode = arch_cfg.get('mode', 'remote')
```

Comment:
Heh, alredy defaulting it twice.  Yep, at least make this get set in one place with `.setdefault()` or such after loading.  Though...  we load in two different places without a function so maybe we'll have to go fix that separately first.